file-browser/src/components/SearchInput.js

```diff
@@ -5,6 +5,6 @@ import React from 'react';
 export default class SearchInput extends React.Component {
     render() {
         const props = this.props;
-        return (<div style={props.style}>SearchInput</div>)
+        return (<div style={props.style}>SearchInputs</div>)
```

Comment:
> So SearchInputs is just a placeholder and we are going to have a search bar?

Yes

> I've made a search bar to search and display videos with Youtube API

Oh this is awesome! can you 
1. upload a screenshot/gif
2. show a demo? 

Keep up with the good work! :+1: